### DISCUSSION TITLE
Should the page use the tracking tool

### DISCUSSION THREAD

**Discussion by BartDubois** [origami-network/origami-network.github.io#16] [Q&A]
The fallowing concerns are part of this discussion.
- Should the tracking like Google Analytics should be used? It could improve visibility how the page is progressing and what is the most interesting thing on the page.
- Should it be mandatory or optional? By choosing the level of tracking.
- Should it be displayed on the page? In form of popup or static text? Anyway it is simple web page.

**Comment by mrmattiago** [comment]
Recently, google analytics has been marked as not secure due to EU law, all data should be stored within EU places, interesting what Google will do with this ;)
